draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -239,6 +239,16 @@
     </section>
     
     <section title="QoS Model Design">
+      <t> The overall objective of this model design is to offer a
```

Comment:
Something looks a miss. I have suggested some changes (marked **), check if that looks better:

"The overall objective of this model design is to offer a **base set of modules** that can then be further augmented to provide other **models** such as Policy Based Routing (PBR) that would augment the base modules defined in this draft to define PBR. This document defines one such method called DiffServ. The design also takes into consideration that vendors might want to extend the model to add their own extensions, such as **to extend** Traffic Policy **module**, to provide constructs of policy for different QoS functionalities."

Reply:
Ok. Here is how it reads now:

     ```
      The overall objective of this model design is to offer a
      base set of modules that can then be further augmented to
      provide other models such as Policy Based Routing (PBR) that
      would augment the base modules defined in this draft to define
      PBR. This document defines one such method called DiffServ. The
      design also takes into consideration that vendors might want to
      extend the model to add their own extensions, such as to extend
      Traffic Policy module, to provide constructs of policy for
      different QoS functionalities.
```